### PR TITLE
ADEN-3358 BFAA unit

### PR DESCRIPTION
**Pull request by fraszczakszymon**
## Links

* http://wikia-inc.atlassian.net/browse/ADEN-3358

### CODE REVIEW

File: front/main/app/styles/component/_site-head.scss [hunk 1]
```diff
@@ -6,6 +6,13 @@ $icon-size: 25px;
 	margin-top: $icon-site-head-width;
 }
 
+body.no-scroll {
+	.site-head {
+		top: 0 !important;
+		position: fixed !important;
+	}
+}
```
Comment (fraszczakszymon) [inline front/main/app/styles/component/_site-head.scss]: 🙈

File: front/main/app/styles/module/_ads.scss [hunk 2]
```diff
@@ -18,12 +18,26 @@ $ad-background: rgb(44, 52, 61);
...
 }
 
-.mobile-top-leaderboard .ad-slot {
-	background: $ad-background;
-	min-height: 50px;
-	padding: 10px;
-	position: relative;
-	z-index: $z-ads-top-leaderboard;
+.mobile-top-leaderboard {
+	.ad-slot {
```
Comment (rogatty) [inline front/main/app/styles/module/_ads.scss]: @hakubo would probably want to add a comment about this one :)
(https://github.com/Wikia/mercury/pull/2436/commits/c25200d75540f483d903ff052c8347ed39849ab9)

Comment (fraszczakszymon) [inline front/main/app/styles/module/_ads.scss]: Isn't it the same rule?

Comment (rogatty) [inline front/main/app/styles/module/_ads.scss]: Of course it is, I missed the space there :see_no_evil:

File: front/main/app/styles/component/_site-head.scss [hunk 3]
```diff
@@ -6,6 +6,13 @@ $icon-size: 25px;
 	margin-top: $icon-site-head-width;
 }
 
+body.no-scroll {
+	.site-head {
+		top: 0 !important;
```
Comment (hakubo) [inline front/main/app/styles/component/_site-head.scss]: what options were considered to not use important?

File: front/main/app/styles/module/_ads.scss [hunk 4]
```diff
@@ -18,12 +18,26 @@ $ad-background: rgb(44, 52, 61);
...
+		background: $ad-background;
+		min-height: 50px;
+		padding: 10px;
+		position: relative;
+		z-index: $z-ads-top-leaderboard;
+	}
+
+	&.bfaa-template {
+		position: absolute;
```
Comment (hakubo) [inline front/main/app/styles/module/_ads.scss]: this is top of the page and position absolute is not the best for performance - any ideas how we could avoid that?

File: front/main/app/styles/component/_site-head.scss (hunk 3 above)
Comment (rogatty) [inline front/main/app/styles/component/_site-head.scss]: I think this is being done to move the site-head to the proper place when side-nav is opened. Am I right? If yes, then maybe we should hide the ad slot instead?

File: front/main/app/styles/module/_ads.scss (hunk 4 above)
Comment (rogatty) [inline front/main/app/styles/module/_ads.scss]: Is a separate ad slot an option? Then we could just put it at the top of body.

**Comment by fraszczakszymon** [comment]
OK, I'm closing it now. We have to do it in different way. Sorry and thanks for your time :)

**Pull request closed without merging**